protocol/SignalingProtocol.proto

```diff
@@ -1,4 +1,4 @@
-// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Copyright 2019-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Is this the new way of doing it :P ?

Reply:
good catch. I will change this.  script needs to be updated.